test/parallel/test-stream-pipe-flow.js

```diff
@@ -65,3 +66,25 @@ const { Readable, Writable, PassThrough } = require('stream');
   wrapper.resume();
   wrapper.on('end', common.mustCall());
 }
+
+{
+  const rs = new PassThrough();
```

Comment:
I would prefer if we really only created a Readable stream here, as that makes the purpose a bit clearer imo.

```suggestion
  const rs = new Readable({ read() {} });
```